pipeline/train/spm-vocab.sh

```diff
@@ -11,14 +11,19 @@ test -v MARIAN
 corpus_src=$1
 corpus_trg=$2
 vocab_output=$3
+sample_size=$4
 
 vocab_dir=$(dirname "${vocab_output}")
 mkdir -p "${vocab_dir}"
 
-pigz -dc "${corpus_src}" | shuf -n 10000000 >"${vocab_dir}/data.txt"
-pigz -dc "${corpus_trg}" | shuf -n 10000000 >>"${vocab_dir}/data.txt"
+pigz -dc "${corpus_src}" >"${vocab_dir}/data.txt"
+pigz -dc "${corpus_trg}" >>"${vocab_dir}/data.txt"
 
 "${MARIAN}/spm_train" --bos_id=-1 --eos_id=0 --unk_id=1 --user_defined_symbols="" \
-  --input="${vocab_dir}/data.txt" --model_prefix="${vocab_dir}/vocab" --vocab_size=32000
```

Comment:
--input takes multiple files, no need to concat them first.

Reply:
From github:
```--input: one-sentence-per-line raw corpus file. No need to run tokenizer, normalizer or preprocessor. By default, SentencePiece normalizes the input with Unicode NFKC. You can pass a comma-separated list of files.```
Not sure it works with gzipped corpora as well.

Reply:
ok, fixed, but it doesn't work with compressed files